### PR TITLE
Updated the diagrams for new configuration changes

### PR DESCRIPTION
**Pull request by venkatmahalingam**
Updated the diagrams for new configuration changes

### CODE REVIEW

**Review by glimchb — COMMENTED** [review]
@mardim91 please review

**Comment by venkatmahalingam** [comment]
@mardim91 I'll have to look at why access link case is not working for VLAN50 from host1, for now, I made it as trunk, will fix it.

**Review by glimchb — APPROVED** [review]

File: OPI-EVPN-PoC.png
Comment (mardim91) [inline OPI-EVPN-PoC.png:1]: Question: Do we need to have a symmetric setup between leaf1 and leaf2 when it comes to the Assymetric IRB case in order to test it properly ? I mean to have in both leafs vni20,vni30 and blueVRFs. Not sure just asking @venkatmahalingam

Comment (venkatmahalingam) [reply OPI-EVPN-PoC.png:1]: In this topology, the assumption is internal nodes (e.g leaf1 & leaf2) have all required VLANs, only for the external traffic (via bleaf) we tend to L3VNI.

File: OPI-EVPN-Leaf1-Detailed-View.png
[the remark this answers is not in the document]
Comment (venkatmahalingam) [reply OPI-EVPN-Leaf1-Detailed-View.png:1]: Will update the diagram for vlan40.

File: OPI-EVPN-L3-Symmetric-IRB.png
[the remark this answers is not in the document]
Comment (venkatmahalingam) [reply OPI-EVPN-L3-Symmetric-IRB.png:1]: sure, my bad, will add the tag.

File: OPI-EVPN-PoC.png
Comment (mardim91) [reply OPI-EVPN-PoC.png:1]: ok then understood

File: OPI-EVPN-Leaf1-Detailed-View.png
[the remark this answers is not in the document]
Comment (mardim91) [reply OPI-EVPN-Leaf1-Detailed-View.png:1]: Here the VNI40 is missing

File: OPI-EVPN-PoC.png
Comment (mardim91) [inline OPI-EVPN-PoC.png:1]: Here you are missing the explanation fo the L2VNI 40 usecase.

**Comment by mardim91** [comment]
@venkatmahalingam please add also the detailed picture of the packet flow for the L3VNI usecases for vlan 50 which is an access vlan.

File: OPI-EVPN-L3-Symmetric-IRB.png
[the remark this answers is not in the document]
Comment (venkatmahalingam) [reply OPI-EVPN-L3-Symmetric-IRB.png:1]: yes, we could change it for better clarity.

File: OPI-EVPN-PoC.png
Comment (venkatmahalingam) [reply OPI-EVPN-PoC.png:1]: Sure, will add it.

**Comment by venkatmahalingam** [comment]
Please ignore the commitline failure, will squash the commits once I get go ahead from @mardim91

**Review by mardim91 — APPROVED** [review]
LGTM. If anything seems off we can update again in the future. For now looks ok

**Pull request merged**